### ISSUE TITLE
Course lesson title edit not working in content area

### ISSUE THREAD

**Issue by brylie** [Crowducate/crowducate-platform#109]
# Observed behavior

When editing a course lesson title, via the inline editor in the content area, the changes are not saved.
# Expected behavior

Updates to the course title, made via the content area inline editor, should save to the database.

**Comment by sebbel** [comment]
Which branch are we talking about here? The develop branch?

**Comment by brylie** [comment]
Yep. Basically, editing the lesson title via the sidebar works, but not the main lesson section.

**Comment by brylie** [comment]
It turns out to be a small fix, so I will have a PR shortly.

**Comment by sebbel** [comment]
When I press the "Edit Course" button, nothing happens. 
tested on branch develop
Chrome
Firefox

**Comment by Rahbaran** [comment]
Same here. Tested on develop branch and feature/course-lesson-title-editable.

**Comment by brylie** [comment]
Please open a separate issue for this bug. It may have been resolved recently, but lets track it separately.

On 8 September 2015 23:14:01 EEST, Amir Rahbaran notifications@github.com wrote:

> Same here. Tested on develop branch and
> feature/course-lesson-title-editable.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/Crowducate/crowducate-platform/issues/109#issuecomment-138686700

## 

Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by sebbel** [comment]
Opened #113

**Comment by brylie** [comment]
See PR #114 for a resolution to this issue and #113.
